### ISSUE TITLE
KeyBoxConfig.properties regenerated every docker restart

### ISSUE THREAD

**Issue by sgohl** [garywiz/docker-keybox#5]
I run the docker container via -d, but after editing the KeyBoxConfig.properties and docker restart, the file is being recreated although I took the persistent storage approach.

**Comment by garywiz** [comment]
Yes, that is actually correct behavior.  The KeyBoxConfig.properties file is based upon a template which is generated from environment variables and start-up parameters, which can change from invocation to invocation, even if persistent storage is used.  Persistent storage is mostly to assure that the database, shared keys, and other saved information aren't lost.

However, I've had this request before.  I'll consider some options.
